db/seeds.rb

```diff
@@ -16,8 +16,8 @@
 end
 
 admin = User.find_or_create_by(admin: true) do |user|
-  password = 'PASSWORD'
-  user.email = 'seed-admin@example.com'
+  password = 'someones_password'
+  user.email = 'someone@example.com'
```

Comment:
Why did you change the default e-mail and password?

Reply:
https://github.com/edgi-govdata-archiving/web-monitoring-db/blob/d06e93f80c0326ccb2b47aa7c8d0ff980e293905/app/views/home/index.html.erb#L14-L19

As in, let's just keep these in sync, so it's less noise :)